libraries/AR_Motors/AP_MotorsUGV.cpp

```diff
@@ -110,6 +110,13 @@ const AP_Param::GroupInfo AP_MotorsUGV::var_info[] = {
     // @User: Standard
     AP_GROUPINFO("VEC_ANGLEMAX", 13, AP_MotorsUGV, _vector_angle_max, 0.0f),
 
+    // @Param: ESC_BACKWARD_UNLOCK
+    // @DisplayName: simulate back throttle unlock.
+    // @Description: Simulate press back then release so that ESC backward is unlocked.
+    // @Range: 0 1
```

Comment:
rather than range better to have a `@Values` Can just be 0: disabled and 1: Enabled